### PR TITLE
refactored, ordered by publish_date

### PR DESCRIPTION
**Pull request by mmdotz**
Started over. This works every which way I checked!

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -11,20 +11,22 @@ def index
...
-      @posts = @posts.limit(10)
+      @posts  = @posts.limit(10)
     end
     @posts = @posts.published.recent_first
   end
 
   def show
-    @post = Post.find_by_permalink(params[:id]) || Post.find(params[:id])
+    @post           = Post.find_by_permalink(params[:id]) || Post.find(params[:id])
+    @previous_post  = Post.where("publish_date < ?", @post.publish_date).order(:publish_date).last
```
Comment (jeffreybiles) [inline app/controllers/posts_controller.rb:21]: Nice!

File: app/models/post.rb [hunk 2]
```diff
@@ -15,4 +15,6 @@ def readable_time
       "#{seconds/60}m#{seconds%60}s"
     end
   end
+
```
Comment (jeffreybiles) [inline app/models/post.rb]: No need for the two blank lines

**Comment by jeffreybiles** [comment]
Nice work!  Elegant solution.

Once you remove the extra newlines I'll merge.  I generally use a visual diff inspector (such as gitx) when committing so I can review my code and commit it on a file-by-file basis.

**Comment by mmdotz** [comment]
Ready

**Pull request merged**